apps/api/src/statistics/repositories/statistics.repository.ts

```diff
@@ -131,6 +133,19 @@ export class StatisticsRepository {
     return result;
   }
 
+  async getMostFivePopularCourses(userId: string) {
```

Comment:
**Suggestion:** Consider renaming `getMostFivsPopularCourses` to `getFiveMostPopularCourses`.
(Logical order)


```suggestion
  async getFiveMostPopularCourses(userId: string) {
```